Viewall

apothuku opened the pull request:
# Changes
This PR builds on the inclusion of all county data and multi-search. Initially, users only see the first 10 rows of the table. They can click View All to see all rows, and then Collapse to return to 10 rows.

A few questions on behavior:
1. Row truncation applies to search as well - when there are more than 10 search results, users will have to click View All to see all results. Would we prefer to show all search results regardless?
2. When there are less than 10 search results, the View All button doesn't appear to the user to have any effect when clicked. Should we disable the button when there are less that 10 search results?

apothuku committed:
Add multi-search to table

apothuku committed:
Add all county data

Adds data from every county for the cash-bail-rate and cash-bail-race
tables. Also updates the averages for that data.

apothuku committed:
Fix sorting signed number column

apothuku committed:
Add view all button to tables

Now that we have 67 rows per table, initially truncate the number of
visible rows to 10 and allow the user to see all values by clicking the
"view all" button.

alexwgraves commented on src/css/table.css:
```diff
@@ -67,3 +67,14 @@ td.sorted {
   border-bottom-color: #eee;
 }
+
+.view-all-btn {
+  width: 100%;
+  padding: 1em;
+  margin: .5em 0 2.5em;
+  text-align: center;
+  font-weight: 500;
+  border: 1px #333 solid;
+  cursor: pointer;
+  user-select: none;
+}
```
These don't quite match the design — looks like they should be right up against the bottom of the table, instead of it having some margin. It also looks like the text is bold in the design.

alexwgraves reviewed (commented):
Looking at this in the browser, I'm not liking how the table resizes slightly when the "view all" button is clicked. This is happening because the county with the longest column name is hidden initially. I think the easiest fix for this is to give the county `td` cells a `width` value (looks like `95px` works pretty well, as the longest county results in `93.22px` width).

apothuku committed:
Merge branch 'master' into viewall

apothuku committed:
Add minor style tweaks to match design

- Set the width of the county name cell to cover the longest county name
to avoid the county name column from resizing when long county names
enter the table.
- Removes the margin between the view all button and the table
- Bolds the view all font.

apothuku commented:
Addressed comments
